packit/cli/create_update.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import os
+
+import click
+
+from packit.api import PackitAPI
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception
+from packit.config import pass_config, get_context_settings, get_local_package_config
+
+logger = logging.getLogger(__file__)
+
+
+@click.command("build", context_settings=get_context_settings())
+@click.option(
+    "--dist-git-branch",
+    help="Target branch in dist-git to release into.",
+    default="master",
+)
+@click.option(
+    "--dist-git-path",
+    help="Path to dist-git repo to work in. "
+    "Otherwise clone the repo in a temporary directory.",
+)
+@click.argument(
+    "repo", type=LocalProjectParameter(), default=os.path.abspath(os.path.curdir)
+)
+@pass_config
+@cover_packit_exception
+def build(config, dist_git_path, dist_git_branch, repo):
```

Comment:
I think this file should be part of #139 (and should call `create_update`)

Reply:
oh man, I messed up; thanks for headsup!